docs/gaia/validators/validator-setup.md

```diff
@@ -1,14 +1,14 @@
 # Validator Setup
 
-::: warning Current Testnet
-The current testnet is `gaia-8000`.
+:::
+The current testnet is `gaia-9001`.
```

Comment:
we need a proper strategy to keep this updated

Reply:
Maybe we can link to the testnets repo in the `latest` directory...

Reply:
https://github.com/cosmos/testnets/tree/master/latest